venues/thecvf.com/ECCV/2020/Conference/webfield/secondary_ac.js

```diff
@@ -0,0 +1,1283 @@
+// Constants
+var CONFERENCE_ID = 'thecvf.com/ECCV/2020/Conference';
```

Comment:
the template in the openreview-py has been modified recently, so I think we should commit these changes in the builder and enable the secondary area chair tab only when a variable `var SECONDARY_AREA_CHAIR_NAME = '';` is not empty. Otherwise it is going to be too difficult to deploy this code in the group without loosing the latest changes in the openreview-py repo.

Reply:
Exactly my thoughts. Working on this.